### PR TITLE
[ADF-3739] Fix task_filters_cloud.e2e.ts  tests

### PR DESCRIPTION
**Pull request by rgherghelas**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/pages/adf/dataTablePage.js [hunk 1]
```diff
@@ -298,19 +298,19 @@ var DataTablePage = function (rootElement = element(by.css("adf-datatable"))) {
     };
 
     this.checkContentIsDisplayed = function(content) {
-        var row = by.cssContainingText("span", content);
+        var row = by.cssContainingText("[class='adf-datatable-cell-value']", content);
```
Comment (eromano) [inline e2e/pages/adf/dataTablePage.js]: please do not use the class selector add an automation ID if you need it

File: e2e/process-services-cloud/task_filters_cloud.e2e.ts [hunk 2]
```diff
@@ -88,15 +92,22 @@ describe('Task filters cloud', () => {
 
             tasksCloudDemoPage.completedTasksFilter().clickTaskFilter();
             expect(tasksCloudDemoPage.checkActiveFilterActive()).toBe('Completed Tasks');
+
             tasksCloudDemoPage.taskListCloudComponent().getDataTable().checkContentIsDisplayed(completedTask);
         });
 
-        xit('[C289957] Should display task filter results when task filter is selected', async () => {
+        it('[C289957] Should display task filter results when task filter is selected', async () => {
```
Comment (cristinaj) [inline e2e/process-services-cloud/task_filters_cloud.e2e.ts]: This test can be deleted, also from test rail. This case is already covered by C290009.

File: e2e/process-services-cloud/task_filters_cloud.e2e.ts [hunk 3]
```diff
@@ -88,15 +92,22 @@ describe('Task filters cloud', () => {
...
             await tasksService.init(user, password);
-            let task = await tasksService.createStandaloneTask(myTask, simpleApp);
+            await tasksService.createStandaloneTask(myTask, simpleApp);
 
             tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
             expect(tasksCloudDemoPage.checkActiveFilterActive()).toBe('My Tasks');
+
+            tasksCloudDemoPage.customFilter().clickCustomFilters();
```
Comment (cristinaj) [inline e2e/process-services-cloud/task_filters_cloud.e2e.ts]: Lets remove these steps of selecting a custom filter by just claiming the task in the preconditions.

File: e2e/pages/adf/process_cloud/taskFiltersCloudComponent.ts [hunk 4]
```diff
@@ -49,6 +49,21 @@ export class TaskFiltersCloudComponent {
         return this.filter.click();
     }
 
+    clickCustomFilters() {
```
Comment (cristinaj) [inline e2e/pages/adf/process_cloud/taskFiltersCloudComponent.ts]: let's move these methods from taskFiltersCloudComponent.ts to tasksCloudDemoPage.ts, cause these dropdowns  are part of demo-shell only.

**Pull request merged**